static/app/utils/replays/projectSupportsReplay.spec.tsx

```diff
@@ -34,11 +34,21 @@ describe('projectSupportsReplay & projectCanLinkToReplay', () => {
   );
 
   it.each([
-    'dotnet' as PlatformKey,
-    'java' as PlatformKey,
     'node' as PlatformKey,
     'php' as PlatformKey,
+    'bun' as PlatformKey,
+    'elixir' as PlatformKey,
+  ])('should SUPPORT Backend framework %s', platform => {
+    const project = mockProject(platform);
+    expect(projectSupportsReplay(project)).toBeTruthy();
+    expect(projectCanLinkToReplay(project)).toBeTruthy();
+  });
+
+  it.each([
+    'java' as PlatformKey,
     'rust' as PlatformKey,
+    'python' as PlatformKey,
+    'go-http' as PlatformKey,
```

Comment:
lets also throw in `angularjs` here too, because that's not supported but could look like it is (a technical way to implment that comment i had above)

but i'm nitpicking now

Reply:
there's already a test for `angularjs` above!

Reply:
oh yeah, this is a list of backend frameworks anyway.. oops